rules-reviewed/eap7/eap4and5/jboss-eap4and5to6and7-xml.windup.xml

```diff
@@ -103,8 +102,8 @@
                 <hint issue-display-mode="all" title="Remove class-loading definition" effort="3" category-id="mandatory">
                     <message>
                         <![CDATA[
-                        The `class-loading` and `loader-repository` configuration in `jboss.xml`, `jboss-web.xml` and `jboss-service.xml` were JBoss EAP 4.x & 5.x configuration options for specifying classloader isolation on a deployment.  
```

Comment:
is this double space removal on purpose ? we put it to create a new line after the text.